package.json

```diff
@@ -30,6 +30,13 @@
     "test:validate": "./scripts/tests-validate.sh",
     "test:browser:filter": "vitest --run --coverage false --config vitest.browser.config.mts",
     "test:e2e": "vitest --run --config vitest.node.config.mts $(scripts/tests-find.sh --e2e)",
+    "bench:node": "vitest bench --run --config vitest.node.config.mts $(scripts/tests-find.sh)",
+    "bench:node:filter": "vitest bench --run --config vitest.node.config.mts",
+    "clinic:flame": "clinic flame --autocannon [ / ] -- node packages/${npm_config_package_name}/dist/profile.js",
+    "clinic:bubble": "clinic bubbleprof --autocannon [ -c 5 -a 500 / ] -- node packages/${npm_config_package_name}/dist/profile.js",
+    "clinic:doctor": "clinic doctor --autocannon [ / ] -- node packages/${npm_config_package_name}/dist/profile.js",
+    "clinic:heap": "clinic heapprofiler --autocannon [ / ] -- node packages/${npm_config_package_name}/dist/profile.js",
+    "0x": "0x -o -P 'autocannon localhost:$PORT' -- node packages/${npm_config_package_name}/dist/profile.js",
```

Comment:
Is it worth having all of these packages under the `bench:` prefix for grouping, like we do with `test`, `build`, `lint` etc.

Reply:
They are profiling tools not _necessarily_ benchmarking tools i.e. you would only use them if you suspect that there is a performance regression related to our code.